docs/features/sessions/changing-instance-type.md

```diff
@@ -25,4 +25,6 @@ Currently the following scenarios are supported:
 2. Changing an On-Demand instance to a [Spot Instance](https://docs.grid.ai/features/runs/interruptible-machines#interruptible-machines)
 3. Changing from a Spot Instance to an On-Demand Instance
 
-<note>Upscaling to a GPU instance from a CPU or smaller GPU instance is not yet supported</note>
+:::note
```

Comment:
this is working now. @robert-s-lee @sunitaprakash to chime in if not

Reply:
the feature does work.